prom_sky_con/Makefile

```diff
@@ -0,0 +1,5 @@
+PROG := prom_sky_con
```

Comment:
please rename exporter name and directory to prometheus

Reply:
This exporter may well run on the same machine where prometheus is itself running, so we don't want to give this exporter the same name as the prometheus server. I therefore called it prometheus-skydive-connector. I can change the directory name to prometheus, since this is local to the skydive flow exporter, but not the name of the runnable program.

Reply:
ok